app/views/reasonableExcuseJourneys/other/WhyReturnSubmittedLatePage.scala.html

```diff
@@ -45,11 +46,14 @@
 
   @pageHeader()
 
-  @formHelper(action = postAction) {
+@formHelper(action = postAction) {
     @characterCount(
       form("why-return-submitted-late-text"),
       label = messages(pageTitleAndHeading),
-      hint = None,
+      hint = if(appealType.contains(PenaltyTypeEnum.Late_Payment) || appealType.contains(PenaltyTypeEnum.Additional))
+Some(HtmlContent(messages("otherReason.whyReturnSubmittedLate.lpp.p1")))
```

Comment:
`otherReason.whyReturnSubmittedLate.p1` should be shown when the user is appealing multiple penalties. Not just an LPP. I believe there is a session key to figure this out: https://github.com/hmrc/penalties-appeals-frontend/blob/563f6bbbcd2c6661ff500e3e261d166ae2010c0c/app/utils/SessionKeys.scala#L57

Reply:
updated